cli_reference/developer_cli_odo/odo-release-notes.adoc

```diff
@@ -7,78 +7,67 @@ include::modules/common-attributes.adoc[]
 toc::[]
 
 [id="odo-notable-improvements_{context}"]
-== Notable changes and improvements in `{odo-title}`
-
-* `{odo-title}` now supports Devfile v2.
-
-* `odo create -s2i` now converts an S2I component into a devfile component. When running `odo create --s2i <component-type>` `{odo-title}` now creates a converted Devfile component based on the S2I images of the specified component type.
+== Notable changes and improvements in `{odo-title}` version 2.5.0
+
+// #5238
+* Creates unique routes for each component, using `adler32` hashing
+// #5252
+* Supports additional fields in the devfile for assigning resources:
+** cpuRequest
+** cpuLimit
+** memoryRequest
+** memoryLimit
+// #5276
+* You can now run the `odo delete` command with the `--deploy` flag to remove components that you deployed using the `odo deploy` command: 
 +
-Note that this feature introduces many breaking changes, see xref:../../cli_reference/developer_cli_odo/odo-release-notes.adoc#odo-known-issues_odo-release-notes[Known Issues] to learn more.
-
-* Operator based service is now created on the cluster only after you run `odo push` and not after `odo service create` anymore.
-
-* You can now use the `--container` flag to specify the container you want to attach storage to when running `odo storage create` command. See xref:../../cli_reference/developer_cli_odo/creating_and_deploying_applications_with_odo/working-with-storage.adoc#adding-storage-to-a-specific-container_working-with-storage[Adding storage to a specific container] to learn the details.
-
-* `odo catalog component describe` now returns correct JSON if the same name is used for a component in multiple registries.
-
-* Commands that implement changes directly on a cluster now display a message informing a user that `odo push` is not required.
+[source,terminal]
+----
+$ odo delete --deploy
+----
+// #5237
+* Adds mapping support to the `odo link` command
+// #5279
+* Supports ephemeral volumes using the `ephemeral` field in `volume` components
+// #5270
+* Sets the default answer to `yes` when asking for telemetry opt-in
+// #5260
+* Improves metrics by sending additional telemetry data to the devfile registry
+// #5287
+* Updates the bootstrap image to `registry.access.redhat.com/ocp-tools-4/odo-init-container-rhel8:1.1.11`
+// #5308 
+* The upstream repository is available at link:https://github.com/redhat-developer/odo[]
```

Comment:
Maybe add this in a section like Additional Resource. Normally we do not point to the upstream GH repos. Exceptions are made if they are under the OpenShift/redhat-developer orgs.

Reply:
I can remove this altogether - I was wondering if the repo had moved from openshift to redhat-developer since the last documented release and if that was worthwhile pointing out

Reply:
Since it is now under the supported RH repos, it is worth mentioning, for sure, +1 to that.
I was just wondering about the placement, should it be added as an additional resource to the RN section or in the [Understanding odo section](https://docs.openshift.com/container-platform/4.9/cli_reference/developer_cli_odo/understanding-odo.html), either in the first paragraph or in the AR. I am not sure about putting it as a new feature. 
Probably leave it as is. When we update the Understanding section, let's add it there as well. Can't hurt.